### ISSUE TITLE
pfstats failing to read pfdhcp stats

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#3730]
Getting the following errors:
```
Nov 01 14:55:48 localhost.localdomain pfstats[13002]: t=2018-11-01T14:55:48-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/queues/stats" pid=13002
Nov 01 14:55:56 localhost.localdomain pfstats[13002]: t=2018-11-01T14:55:56-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/dhcp/stats" pid=13002
Nov 01 14:55:56 localhost.localdomain pfstats[13002]: t=2018-11-01T14:55:56-0400 lvl=warn msg="Compile error '$.items[*].network.free, $.items[*].free' parse error from GET /api/v1/dhcp/stats: Expected Type to be a Map." pid=13002
Nov 01 14:55:56 localhost.localdomain pfstats[13002]: t=2018-11-01T14:55:56-0400 lvl=warn msg="Unhandled response type from GET /api/v1/dhcp/stats" pid=13002
Nov 01 14:56:02 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:02-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/queues/stats" pid=13002
Nov 01 14:56:03 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:03-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/dhcp/stats" pid=13002
Nov 01 14:56:03 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:03-0400 lvl=warn msg="Compile error '$.items[*].network, $.items[*].percentfree' parse error from GET /api/v1/dhcp/stats: Expected Type to be a Map." pid=13002
Nov 01 14:56:03 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:03-0400 lvl=warn msg="Unhandled response type from GET /api/v1/dhcp/stats" pid=13002
Nov 01 14:56:28 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:28-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/dhcp/stats" pid=13002
Nov 01 14:56:28 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:28-0400 lvl=warn msg="Compile error '$.items[*].network.used, $.items[*].used' parse error from GET /api/v1/dhcp/stats: Expected Type to be a Map." pid=13002
Nov 01 14:56:28 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:28-0400 lvl=warn msg="Unhandled response type from GET /api/v1/dhcp/stats" pid=13002
Nov 01 14:56:32 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:32-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/dhcp/stats" pid=13002
Nov 01 14:56:32 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:32-0400 lvl=warn msg="Compile error '$.items[*].network, $.items[*].percentused' parse error from GET /api/v1/dhcp/stats: Expected Type to be a Map." pid=13002
Nov 01 14:56:32 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:32-0400 lvl=warn msg="Unhandled response type from GET /api/v1/dhcp/stats" pid=13002
Nov 01 14:56:42 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:42-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/queues/stats" pid=13002
Nov 01 14:56:48 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:48-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/queues/stats" pid=13002
Nov 01 14:56:56 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:56-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/dhcp/stats" pid=13002
Nov 01 14:56:56 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:56-0400 lvl=warn msg="Compile error '$.items[*].network.free, $.items[*].free' parse error from GET /api/v1/dhcp/stats: Expected Type to be a Map." pid=13002
Nov 01 14:56:56 localhost.localdomain pfstats[13002]: t=2018-11-01T14:56:56-0400 lvl=warn msg="Unhandled response type from GET /api/v1/dhcp/stats" pid=13002
Nov 01 14:57:02 localhost.localdomain pfstats[13002]: t=2018-11-01T14:57:02-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/queues/stats" pid=13002
Nov 01 14:57:03 localhost.localdomain pfstats[13002]: t=2018-11-01T14:57:03-0400 lvl=info msg="Calling Unified API on uri: https://127.0.0.1:9999/api/v1/dhcp/stats" pid=13002
Nov 01 14:57:03 localhost.localdomain pfstats[13002]: t=2018-11-01T14:57:03-0400 lvl=warn msg="Compile error '$.items[*].network, $.items[*].percentfree' parse error from GET /api/v1/dhcp/stats: Expected Type to be a Map." pid=13002
Nov 01 14:57:03 localhost.localdomain pfstats[13002]: t=2018-11-01T14:57:03-0400 lvl=warn msg="Unhandled response type from GET /api/v1/dhcp/stats" pid=13002
```

**Comment by fdurand** [comment]
Tested on latest devel and not able to reproduce.

**Comment by julsemaan** [comment]
I have a routed L3 network, is it possible that causes the issue ?

**Comment by fdurand** [comment]
Can you do that:
curl http://127.0.0.1:22222/api/v1/dhcp/stats

**Comment by julsemaan** [comment]
```
[
    {
        "category": "none",
        "free": 237,
        "interface": "ens224",
        "members": null,
        "network": "172.21.156.0/24",
        "options": {
            "optionDomainName": "vlan-registration.example.com",
            "optionDomainNameServer": "172.21.156.11,172.21.156.12",
            "optionIPAddressLeaseTime": "30s",
            "optionRouter": "172.21.156.12,172.21.156.11",
            "optionSubnetMask": "255.255.255.0"
        },
        "status": "Normal"
    },
    {
        "category": "none",
        "free": 235,
        "interface": "ens224",
        "members": null,
        "network": "172.21.158.0/24",
        "options": {
            "optionDomainName": "vlan-registration.example.com",
            "optionDomainNameServer": "172.21.156.11,172.21.156.12",
            "optionIPAddressLeaseTime": "30s",
            "optionRouter": "172.21.158.1",
            "optionSubnetMask": "255.255.255.0"
        },
        "status": "Normal"
    }
]
```

**Comment by julsemaan** [comment]
Was actually my codebase that was too old because of the server running the pfdhcp with the multi-zone fixes

**Comment by nqb** [comment]
I get same messages with `packetfence-8.1.9-0.201811020000.el7` on CentOS 7 with a standalone setup.

**Comment by nqb** [comment]
```
# curl http://127.0.0.1:22222/api/v1/dhcp/stats
{"items":null,"status":"200"}
```

**Comment by fdurand** [comment]
Fixes by 2e68f74669a65626bc823c4e51e3b3aa473bee83
